### DISCUSSION TITLE
Feature request: post-competition export shot identification

### DISCUSSION THREAD

**Discussion by amplejoe** [lucaro/DRES#279] [Ideas]
It is difficult to precisely identify the targeted media item for visual/textual KIS tasks when using the exported query set in JSON format, as is made available for VBS2021 [here](https://videobrowsershowdown.org/call-for-papers/vbs-server/). The problem is that the server-generated media item ids can not easily be mapped to the underlying videos, making it hard to find a particular target segment.

It would be great to include some information to make this easier, i.e. a field containing the filename/videoId/(shotIds). This would much ease retrieving post-competition information on particular tasks, e.g. for describing task solution strategies in papers.

Thank you!

**Comment by sauterl** [comment]
Thank you for your input. Indeed, the file available from VBS2021 is not that usable in this regard. This type of export is no longer available (see #255 ), as it was meant to be used within the server only. However, there already exists proper export functionality for the CLI (`compeition export -h` for further information).

Anyways, we are working towards making exports for the competition available (#281 )

**Reply by amplejoe** [reply]
I see - thank you for this clarification!

**Comment by sauterl** [comment]
With the next release of DRES, this behaviour will be added (as #281  got quickly taken care of). With regards to the VBS2021 competition, I guess one could ask the organisers for the proper competition export.

EDIT: @amplejoe It would be kind if you could mark any of the answers as solution, so that people can see that this discussion was resolved 😃

**Reply by amplejoe** [reply]
I would if I had the option to do that.

**Reply by sauterl** [reply]
Wouldn't you (as OP) do that by voting (pressing the arrow up button) on the reply that satisfies your question the most?

**Comment by klschoef** [comment]
Unfortunately, at Klagenfurt University we used temporally virtual server for the VBS, which was shut down (removed) a few days after the VBS. The entire data directory was copied/exported though (data, events, logs, statistics), but it seems there is no possibility to extract the video numbers from this data (only the GUID, which we cannot link back) - or is there one?

**Reply by lucaro** [reply]
DRES stores all its data in the (aptly named) 'data' directory, so as long as you have that one, all run information can still be restored. There was a difference between the export available via the frontend and the one available via the CLI, where the former only contained the IDs. There is a backend export of the entire run though, which also contains the media information as well as the submissions, see: https://drive.google.com/file/d/1Ny7aXsHuLaN0ioJX2JzRrjFN-EjRD3RQ/view?usp=sharing

**Comment by klschoef** [comment]
Thanks very much, Luca! It is good to know that every information is there (if someone needs the mapping between GUID and video ID, please just let me know). I think we can close this thread now.
